example of while loop.

```diff
@@ -0,0 +1,13 @@
+#include<stdio.h>
+void main()
+{
+     int i;
+     i=1 ;
+     while(i<=5)
+     {
+          printf("computer classes\n");
+          i = i+1;
+     }
+     print("learning c");
```

Comment:
Use printf instead of print.

Reply:
Anchal,Just go through the little change that I have done after merging i.e use of printf in place of print.